tools/flecstan/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# -*- CMake -*-
+
+project(flecstan LANGUAGES CXX)
+cmake_minimum_required(VERSION 3.8)
+
+# use cmake modules from cinch
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../cinch/cmake)
+
+# C++
+set(CMAKE_CXX_COMPILER g++)
```

Comment:
That is a bit of a no-no. How would a user tell CMake to use `clang++` or g++-9`? Please remove.